linux_os/guide/system/software/gnome/dconf_use_text_backend/oval/shared.xml

```diff
@@ -0,0 +1,37 @@
+<def-group>
+  <definition class="compliance" id="dconf_use_text_backend" version="1">
+    <metadata>
+      <title>{{{ full_name }}}</title>
```

Comment:
`full_name` translates to the full name of the product, eg. `Red Hat Enterprise Linux 7". It doesn't look like a right definition title.

Reply:
Good catch.